src/test/scala/com/amazon/deequ/KLL/KLLDistanceTest.scala

```diff
@@ -73,4 +74,76 @@ class KLLDistanceTest extends WordSpec with Matchers with SparkContextSpec
       "f" -> 11L, "a" -> 20L, "c" -> 25L, "d" -> 10L, "e" -> 10L)
     assert(Distance.categoricalDistance(sample1, sample2) == 0.0)
   }
+
+
+  // Tests using chi-square method for categorical variables
+
+  "Categorial distance should compute correct chisquare stats with missing bin values" in {
```

Comment:
nit: `... with missing bin values and correcting for low samples`

Reply:
corrected